### ISSUE TITLE
Unable to use bbr-uaadb job with `bosh create-env`

### ISSUE THREAD

**Issue by rkoster** [cloudfoundry/uaa-release#85]
Unable to use bbr-uaadb with `bosh create-env` because the backup [template assumes a bosh link](release_level_backup) which are not supported.

Would it be possible to backup by default, and only skip backups if links is available and `release_level_backup` is set to false?

**Comment by DennisDenuto** [comment]
Thanks for creating this issue. Your suggestion sounds good to me. 

@sreetummidi This "bug" should really only affect operators spinning up a new bosh environment with `uaa` and `bbr-uaadb`. (Or theoretically an operator trying to `create-env` a standalone `uaa` with `bbr-uaadb`)

Furthermore, this approach will not allow disabling `bbr-uaadb` in the context of a `bosh create-env`.

**Comment by JamesClonk** [comment]
Any update on this issue?

I have the same problem when trying to `create-env` a bosh with `bbr-uaadb` in it (which I need for backups!):
```
Deploying:
  Building state for instance 'bosh/0':
    Rendering job templates for instance 'bosh/0':
      Rendering templates for job 'bbr-uaadb/3a3b455d3de6d7f3e5196f5fe8efe63f1b18b160':
        Rendering template src: backup.sh.erb, dst: bin/bbr/backup:
          Rendering template src: /root/.bosh/installations/b6c24fff-8ac4-4ad1-7d1e-2d6081f7552d/tmp/bosh-release-job728798085/templates/backup.sh.erb, dst: /root/.bosh/installations/b6c24fff-8ac4-4ad1-7d1e-2d6081f7552d/tmp/rendered-jobs07
4502660/bin/bbr/backup:
            Running ruby to render templates:
              Running command: 'ruby /root/.bosh/installations/b6c24fff-8ac4-4ad1-7d1e-2d6081f7552d/tmp/erb-renderer040978390/erb-render.rb /root/.bosh/installations/b6c24fff-8ac4-4ad1-7d1e-2d6081f7552d/tmp/erb-renderer040978390/erb-contex
t.json /root/.bosh/installations/b6c24fff-8ac4-4ad1-7d1e-2d6081f7552d/tmp/bosh-release-job728798085/templates/backup.sh.erb /root/.bosh/installations/b6c24fff-8ac4-4ad1-7d1e-2d6081f7552d/tmp/rendered-jobs074502660/bin/bbr/backup', stdout:
'', stderr: '/root/.bosh/installations/b6c24fff-8ac4-4ad1-7d1e-2d6081f7552d/tmp/erb-renderer040978390/erb-render.rb:189:in `rescue in render': Error filling in template '/root/.bosh/installations/b6c24fff-8ac4-4ad1-7d1e-2d6081f7552d/tmp/bo
sh-release-job728798085/templates/backup.sh.erb' for bbr-uaadb/0 (line 11: #<NoMethodError: undefined method `link' for #<TemplateEvaluationContext:0x0000000120c898>>) (RuntimeError)
        from /root/.bosh/installations/b6c24fff-8ac4-4ad1-7d1e-2d6081f7552d/tmp/erb-renderer040978390/erb-render.rb:175:in `render'
        from /root/.bosh/installations/b6c24fff-8ac4-4ad1-7d1e-2d6081f7552d/tmp/erb-renderer040978390/erb-render.rb:200:in `<main>'
```

**Comment by andrewedstrom** [comment]
This came from a reliance on bosh links, which are not supported in `create-env`. It has been fixed in https://www.pivotaltracker.com/story/show/163156938

**Comment by mmurphy30** [comment]
What was the fix for this issue?   I am having this problem now.

**Comment by andrewedstrom** [comment]
@mmurphy30 What UAA version are you on? It was fixed in v74.0.0

The fix was to stop relying on Bosh links and use bosh properties instead.
